annotation/annotation/tasks/services.py

```diff
@@ -62,6 +62,33 @@
     ValidationSchema,
 )
 
+
+class LRU(OrderedDict):
+    def __init__(self, capacity):
+        self.capacity = capacity
+
+    def __contains__(self, item):
+        if super().__contains__(item):
+            self.move_to_end(item)
+            return True
+        else:
+            return False
+
+    def __getitem__(self, key):
+        if key in self:
+            self.move_to_end(key)
+            return super().__getitem__(key)
```

Comment:
```suggestion
            return self[key]
```

Reply:
but better solution would be first get the item first (if it is not found - exception is raised anyway, then move to end and return the found item. More pythonic approach. Otherwise you do extra work and raise exception without any details